### DISCUSSION TITLE
SPIRAM initializing error

### DISCUSSION THREAD

**Discussion by Mowitra** [Mair/esp32-course#151] [Course Qeustions]
In large json video I followed the steps as you told but I got this SPIRAM initializing problem.

ets Jun  8 2016 00:22:57

rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
configsip: 0, SPIWP:0xee
clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
mode:DIO, clock div:2
load:0x3fff0030,len:6608
load:0x40078000,len:14788
ho 0 tail 12 room 4
load:0x40080400,len:3792
0x40080400: _init at ??:?

entry 0x40080694
I (28) boot: ESP-IDF v4.4.1 2nd stage bootloader
I (29) boot: compile time 15:35:44
I (29) boot: chip revision: 1
I (32) boot_comm: chip revision: 1, min. bootloader chip revision: 0
I (39) boot.esp32: SPI Speed      : 40MHz
I (43) boot.esp32: SPI Mode       : DIO
I (48) boot.esp32: SPI Flash Size : 4MB
I (53) boot: Enabling RNG early entropy source...
I (58) boot: Partition Table:
I (61) boot: ## Label            Usage          Type ST Offset   Length
I (69) boot:  0 nvs              WiFi data        01 02 00009000 00006000
I (76) boot:  1 phy_init         RF data          01 01 0000f000 00001000
I (84) boot:  2 factory          factory app      00 00 00010000 00100000
I (91) boot: End of partition table
I (95) boot_comm: chip revision: 1, min. application chip revision: 0
I (102) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=1efc8h (126920) map
I (157) esp_image: segment 1: paddr=0002eff0 vaddr=3ffb0000 size=01028h (  4136) load
I (159) esp_image: segment 2: paddr=00030020 vaddr=400d0020 size=97d10h (621840) map
I (388) esp_image: segment 3: paddr=000c7d38 vaddr=3ffb1028 size=028b8h ( 10424) load
I (392) esp_image: segment 4: paddr=000ca5f8 vaddr=40080000 size=1878ch (100236) load
I (435) esp_image: segment 5: paddr=000e2d8c vaddr=50000000 size=00010h (    16) load
I (447) boot: Loaded app from partition at offset 0x10000
I (447) boot: Disabling RNG early entropy source...
I (459) psram: This chip is ESP32-D0WD
**E (460) psram: PSRAM ID read error: 0xffffffff
E (461) cpu_start: Failed to init external RAM!**

abort() was called at PC 0x40081524 on core 0
0x40081524: call_start_cpu0 at C:/ESP_IDF/esp-idf/components/esp_system/port/cpu_start.c:410 (discriminator 3)

Backtrace:0x40081e7d:0x3ffe3ba00x4008c439:0x3ffe3bc0 0x4009320e:0x3ffe3be0 0x40081524:0x3ffe3c50 0x400794dc:0x3ffe3c80  |<-CORRUPTED
0x40081e7d: panic_abort at C:/ESP_IDF/esp-idf/components/esp_system/panic.c:402

0x4008c439: esp_system_abort at C:/ESP_IDF/esp-idf/components/esp_system/esp_system.c:128

0x4009320e: abort at C:/ESP_IDF/esp-idf/components/newlib/abort.c:46

0x40081524: call_start_cpu0 at C:/ESP_IDF/esp-idf/components/esp_system/port/cpu_start.c:410 (discriminator 3)

ELF file SHA256: 84d6ac608440655d

**Comment by Mair** [comment]
Are you sure the chip you have has PSRAM?
see https://learnesp32.com/videos/7/4_psram
